modules/common/security/apparmor/default.nix

```diff
@@ -0,0 +1,30 @@
+# Copyright 2024-2025 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+{
+  config,
+  lib,
+  ...
+}: let
+  cfg = config.ghaf.security.apparmor;
+in {
+  ## Option to enable Apparmor security
+  options.ghaf.security.apparmor = {
+    enable = lib.mkOption {
+      description = ''
+        Enable Apparmor security.
+      '';
+      type = lib.types.bool;
+      default = false;
+    };
+  };
+
+  imports = [
+    ./profiles/firefox.nix
+    ./profiles/chromium.nix
+  ];
+
+  config = lib.mkIf cfg.enable {
+    security.apparmor.enable = true;
+    security.apparmor.killUnconfinedConfinables = lib.mkDefault true;
```

Comment:
I was always a bit skeptical about this option when it was added to NixOS. Does firefox/chromium properly restarts itself afterwards?